### PR TITLE
Feat_FE_reviews

### PR DESCRIPTION
**Pull request by DiamondPurvis**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status
- [ ] Complete, tested, ready to review and merge
- [x ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: client/src/components/Navigation/index2.jsx [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    <ul>
+        <RoutedButton label="Sign In" path={ROUTES.SIGN_IN} />
+        <RoutedButton label="Sign Up" path={ROUTES.SIGN_UP} />
+    </ul>
+    </Box>
+  
+  );
+  
+  export default () => {
+    const [screen, setScreen] = React.useState(4)
```
Comment (CLantigua2) [inline client/src/components/Navigation/index2.jsx]: what is the purpose of putting the nav component into a hash table then passing it into this component? If you're trying to conditionally render between the regular nav2 and authNav, there is a much better way to do this then writing separate components. Please clean this up.

File: client/src/components/Styling/Theme.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        radius: "3px"
+      }
+    },
+    input: {
+      border: {
+        radius: "16px"
+      }
+    },
+    font: {
+      family: "'Roboto', sans-serif",
```
Comment (CLantigua2) [inline client/src/components/Styling/Theme.js]: why is this here again when you already have it in the other component?

File: client/src/components/Navigation/index2.jsx [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
...ts.gstatic.com/s/roboto/v20/KFOmCnqEu92Fr1Mu4WxKOzY.woff2) format('woff2');\n  unicode-range: U+0370-03FF;\n}\n/* vietnamese */\n@font-face {\n  font-family: 'Roboto';\n  font-style: normal;\n  font-weight: 400;\n  src: local('Roboto'), local('Roboto-Regular'), url(https://fonts.gstatic.com/s/roboto/v20/KFOmCnqEu92Fr1Mu7WxKOzY.woff2) format('woff2');\n  unicode-range: U+0102-0103, U+0110-0111, U+1EA0-1EF9, U+20AB;\n}\n/* latin-ext */\n@font-face {\n  font-family: 'Roboto';\n  font-style: normal;\n  font-weight: 400;\n  src: local('Roboto'), local('Roboto-Regular'), url(https://fonts.gstatic.com/s/roboto/v20/KFOmCnqEu92Fr1Mu7GxKOzY.woff2) format('woff2');\n  unicode-range: U+0100-024F, U+0259, U+1E00-1EFF, U+2020, U+20A0-20AB, U+20AD-20CF, U+2113, U+2C60-2C7F, U+A720-A7FF;\n}\n/* latin */\n@font-face {\n  font-family: 'Roboto';\n  font-style: normal;\n  font-weight: 400;\n  src: local('Roboto'), local('Roboto-Regular'), url(https://fonts.gstatic.com/s/roboto/v20/KFOmCnqEu92Fr1Mu4mxK.woff2) format('woff2');\n  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;\n}\n"
```
[the remark this answers is not in the document]
Comment (DiamondPurvis) [reply client/src/components/Navigation/index2.jsx]: this is custom theme for nav bar with grommet, which is different from the landing page on the current code base

File: client/src/components/Styling/Theme.js (hunk 2 above)
Comment (DiamondPurvis) [reply client/src/components/Styling/Theme.js]: I was trying to find a better way to implement the custom theme, it is still a work in progress

File: client/src/App.js [hunk 4]
```diff
@@ -18,25 +18,113 @@ import './App.css';
...
+import Review from './views/Review';
 
 //@ utils
 import * as ROUTES from './Routes/routes';
 
+//@theme
+import Theme from './components/Styling/Theme';
 
-const theme = {
+const hivestack =
```
Comment (Memitaru) [inline client/src/App.js]: Is this the same theme that's in index2.js and theme.js? At first glance they look identical. If so I'm sure the one from theme.js could be imported 👍

File: client/src/components/Navigation/index2.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (DiamondPurvis) [reply client/src/components/Navigation/index2.jsx]: It is an entirely different navbar from the landing page navbar in the current code base. It also uses a new custom theme from grommet. Thought I removed the face under family, guess I missed that.

File: client/src/components/Styling/Theme.js (hunk 2 above)
Comment (DiamondPurvis) [reply client/src/components/Styling/Theme.js]: The theme styling was created to reduce duplicate code blocks with the custom theme, still working on implementing that. Face has also been removed.

File: client/src/App.js (hunk 4 above)
Comment (DiamondPurvis) [reply client/src/App.js]: Yes, I tried this. It's not working at the moment, which is why I have them in all files that needs it until I can fix it.
